packages/story-editor/src/components/checklist/videoOptimizationCheckbox/index.js

```diff
@@ -76,7 +78,7 @@ function VideoOptimizationCheckbox() {
   const checked = currentUser?.meta?.web_stories_media_optimization;
   const showCheckbox = !checked && hasUploadMediaAction;
 
-  return showCheckbox ? (
+  return showCheckbox && isChecklistMounted ? (
```

Comment:
💭 `showCheckbox` is defined a little higher ups o we could combine those checks (if we wanted to make this one variable). Not really a big deal.